src/styles/getReportActionContextMenuStyles.js

```diff
@@ -25,10 +25,11 @@ const bigWrapperStyle = [
  * Generate the wrapper styles for the ReportActionContextMenu.
  *
  * @param {Boolean} isMini
+ * @param {Boolean} isSmallScreenWidth
  * @returns {Array}
  */
-function getReportActionContextMenuStyles(isMini) {
-    return isMini ? miniWrapperStyle : bigWrapperStyle;
+function getReportActionContextMenuStyles(isMini, isSmallScreenWidth) {
+    return isMini ? miniWrapperStyle : [...bigWrapperStyle, !isSmallScreenWidth && styles.pv3];
```

Comment:
```suggestion
    return isMini ? miniWrapperStyle : [...bigWrapperStyle, isSmallScreenWidth ? {} : styles.pv3];
```

just to have the array contains objects and not a mix of objects and boolean